docs/userguide/scheduling/cluster-resources.md

```diff
@@ -104,10 +103,11 @@ Considering the amount of available resources, the scheduler prefers to schedule
 #### Background
 
 `ResourceSummary` describes the overall available resources of the cluster.
-However, the `ResourceSummary` is not precise enough, it mechanically counts the resources on all nodes, but ignores the fragment resources. For example, a cluster with 2000 nodes, 1 core CPU left on each node.
-From the `ResourceSummary`, we get that there are 2000 core CPU left for the cluster, but actually, this cluster cannot run any pod that requires CPU greater than 1 core.
+However, the `ResourceSummary` is not precise enough, it mechanically counts the resources on all nodes, but ignores the fragmented resources of these nodes. For example, a cluster with 2000 nodes has only 1 core CPU left on each node.
+From the `ResourceSummary`, we get that there are 2000 cores CPU left for the cluster, but actually, this cluster cannot run any pod that requires more than 1 core CPU.
 
-Therefore, we introduce a `CustomizedClusterResourceModeling` for each cluster that records the resource portrait of each node. Karmada will collect node and pod information for each cluster. After calculation, this node will be divided into the appropriate resource model configured by the users.
+Therefore, we introduce a `CustomizedClusterResourceModeling` for each cluster that records the resource profile of each node.
+Karmada collects node and pod information from each cluster and computes the appropriate -user configured- resource model to categorize the node into.
```

Comment:
```suggestion
Karmada collects node and pod information from each cluster and computes the appropriate `user configured` resource model to categorize the node info.
```
Is it a typo here?

Reply:
It was meant to say that a node is being categorized into a resource model.

Reply:
Oh. I get it. Thanks.